headers/extract.py

```diff
@@ -43,7 +43,7 @@ def split_class_structure(version, contents):
             else:
                 cctor_thread_padding = False
             break
-        if '*' in ftype or ftype in ('size_t',):
+        if '*' in ftype or ftype in ('size_t', 'Il2CppMetadataTypeHandle'):
```

Comment:
Just to check - is `Il2CppMetadataTypeHandle` pointer-sized (i.e. 32-bit on 32-bit systems and 64-bit on 64-bit systems)? I should have added a comment near here stating that the first if-test was for pointer-sized fields, and the second test (elif) was for fields that were always 32-bit.

Reply:
Yes all 5 of the new types are `size-t`s :)